makefile

```diff
@@ -73,11 +74,16 @@ medgen-disease-extract.owl: medgen-disease-extract.obo
 	owltools $< -o $@
 
 # SSSOM ----------------------------------
-medgen.obographs.json:
-	robot convert -i medgen-disease-extract.owl -o $@
-
-medgen.sssom.tsv: medgen.obographs.json
-	sssom parse medgen.obographs.json -I obographs-json -m config/medgen.sssom-metadata.yml -o $@
+# todo: comemented out old pipeline: remove
+#medgen.obographs.json:
+#	robot convert -i medgen-disease-extract.owl -o $@
+#
+#medgen.sssom.tsv: medgen.obographs.json
+#	sssom parse medgen.obographs.json -I obographs-json -m config/medgen.sssom-metadata.yml -o $@
+sssom: hpo-umls.sssom.tsv
+
+hpo-umls.sssom.tsv hpo-mesh.sssom.tsv: ftp.ncbi.nlm.nih.gov/pub/medgen/MedGenIDMappings.txt
+	python src/create_sssom.py --input-mappings $< --input-sssom-config config/medgen.sssom-metadata.yml
```

Comment:
## Make goal(s)
- [ ] @twhetzel @matentzn review
I added a multi-target goal for these 2 files, added a `.PHONY` `sssom` command for them, and added `sssom` to the build command.

We had some previous, unused SSSOM-related stuff from Chris's original make/Perl based pipeline, but it's not used, so I commented it out. We may just want to remove all that stuff now, as it's littering the `makefile`, unused, making it harder to understand what's going on.

Reply:
Sounds good. I'll remove Chris's old code at some point. Made a low priority issue for it:
- #21